openverse_catalog/dags/providers/provider_workflows.py

```diff
@@ -178,8 +179,9 @@ def __post_init__(self):
     ),
     ProviderWorkflow(
         provider_script="phylopic",
+        ingestion_callable=PhylopicDataIngester,
```

Comment:
Do we know what's the actual start_date for Phylopic images? I've run this for several days in 2011 and got no return. Does it make sense to update it closer to the actual start date for Phylopic to prevent unnecessary requests?

Reply:
When this was weekly, it would only take a few runs to actually pull data (see the durations [in this screenshot above](https://github.com/WordPress/openverse-catalog/pull/747#issuecomment-1287482934) as a proxy for when data starts coming in). We could bump it up to that week I suppose!